### PR TITLE
Replaced calculator's division and multiplication symbol.

### LINKED ISSUE #2751: [FEATURE] make calculator symbols more universal
### Please confirm the following:

- [X] I've checked [the current issues](https://github.com/Ivy-Apps/ivy-wallet/issues) for duplicate issues.
- [X] I've requested a single (only one) feature/change in this issue. It complies with the One Request Per GitHub Issue (ORPGI) rule.
- [X] My issue is well-defined and describes how it should be implemented from UI/UX perspective.

### What do you want to be added or improved?

As a user I want to have the selected calculator symbols to match universal layout of the classic calculator.

### Why do you need it?

Almost every calculator layout looks like the one below, and it's the most universal way to display highlighted mathematical symbols.

### How do you imagine it?

Replace:
`/` with `÷`
`*` with `×`

### PR DESCRIPTION
**Pull request by amitYadavDev**
## Pull Request (PR) Checklist
Please check if your pull request fulfills the following requirements:
- [x] The PR is submitted to the `main` branch.
- [x] I've read the [Contribution Guidelines](https://github.com/Ivy-Apps/ivy-wallet/blob/main/CONTRIBUTING.md).
- [x] I've read the [Architecture Guidelines](https://github.com/Ivy-Apps/ivy-wallet/blob/main/docs/Architecture.md).
- [x] My code builds and is tested on a real Android device.
- [x] I confirm that I've run the code locally and everything works as expected.

## What's changed?

Describe with a few bullets **what's new:**
- a
- b
- c
- d

_💡 Tip: Please, attach screenshots and screen recordings. It helps a lot!_

## Risk Factors

**What may go wrong if we merge your PR?**

- a
- b
- c

**In what cases your code won't work?**

- a
- b
- c

## Does this PR closes any GitHub Issues?

Check **[Ivy Wallet Issues](https://github.com/Ivy-Apps/ivy-wallet/issues)**.

- Closes #2751 

_Replace `{2751}` with the id/number of the issue that you've fixed._

### CODE REVIEW

**Comment by amitYadavDev** [comment]
> The code looks good but I believe it'll have the same problem as your last PR. Can you please upload a screen recording using the calculator?

Yeah sure @ILIYANGERMANOV, but I need some time.

**Comment by ILIYANGERMANOV** [comment]
> > The code looks good but I believe it'll have the same problem as your last PR. Can you please upload a screen recording using the calculator?
> 
> Yeah sure @ILIYANGERMANOV, but I need some time.
> 

No worries, take your time. No rush :)

**Comment by amitYadavDev** [comment]
Hey @ILIYANGERMANOV, Keval library is not supporting (÷, x) these symbols.

**Comment by ILIYANGERMANOV** [comment]
> Hey @ILIYANGERMANOV, Keval library is not supporting (÷, x) these symbols. Look at the code. Screenshot (387)

Exactly, my point. That's why you need to make:
- change only the button labels to the fancy symbols
- use the "*" and "/" for the string

**Comment by amitYadavDev** [comment]
yeah @ILIYANGERMANOV, issue resolved see the video.

https://github.com/Ivy-Apps/ivy-wallet/assets/45551012/5ab3c285-d059-40c6-8ae1-a10db8ab3309

**Comment by amitYadavDev** [comment]
It seems, I need to close this PR and create new one.

File: temp-legacy-code/src/main/java/com/ivy/legacy/legacy/ui/theme/modal/edit/CalculatorModal.kt [hunk 1]
```diff
@@ -215,7 +216,15 @@ private fun formatExpression(expression: String, currency: String): String {
 private fun calculate(expression: String): Double? {
     return try {
         // Keval doesn't support negative numbers, so we add a zero in front of the expression
-        val modifiedExpression = if (expression.startsWith("-")) "0$expression" else expression
+        var tempExpression = ""
+        for(exp in expression) {
+            when (exp) {
```
Comment (ILIYANGERMANOV) [inline temp-legacy-code/src/main/java/com/ivy/legacy/legacy/ui/theme/modal/edit/CalculatorModal.kt:221]: Nice work! Can you use `buildString { }` so its more efficient and more idiomatic. 
```kotlin
val expression = buildString {
   for(char in expression) {
      // your code
   }
}
```

**Review by ILIYANGERMANOV — COMMENTED** [review]
Looks good! Added a comment for minor refactoring and optimization @amitYadavDev

**Comment by ILIYANGERMANOV** [comment]
@amitYadavDev you also need to fix Detekt errors:

/home/runner/work/ivy-wallet/ivy-wallet/temp-legacy-code/src/main/java/com/ivy/legacy/legacy/ui/theme/modal/edit/CalculatorModal.kt:220:9: Missing spacing after "for" [SpacingAroundKeyword]
/home/runner/work/ivy-wallet/ivy-wallet/temp-legacy-code/src/main/java/com/ivy/legacy/legacy/ui/theme/modal/edit/CalculatorModal.kt:1:1: Unused import [NoUnusedImports]

**Comment by amitYadavDev** [comment]
@ILIYANGERMANOV pls look into the code, can I push it?
Screenshot (388)

**Comment by amitYadavDev** [comment]
@ILIYANGERMANOV will create new PR.

**Pull request closed without merging**